app/models/garden.rb

```diff
@@ -2,4 +2,8 @@ class Garden < ApplicationRecord
   validates_presence_of :name, :max_plant_capacity
 
   has_many :plants, :dependent => :destroy
+
```

Comment:
This makes a lot more sense and that's what my mentor was saying. Fat Models, skinny controllers.